<ui version="4.0" >
 <author>Roberto Pariset</author>
 <class>AutoReplacePrefsUI</class>
 <widget class="QWidget" name="AutoReplacePrefsUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Sentence Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="DotEndSentence" >
        <property name="text" >
         <string>Add a dot at the end of each sent line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CapitalizeBeginningSentence" >
        <property name="text" >
         <string>Start each sent line with a capital letter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Replacements List</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QTreeWidget" name="m_list" >
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_add" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_edit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_remove" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>&amp;Text:</string>
          </property>
          <property name="buddy" >
           <cstring>m_key</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_key" />
        </item>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Re&amp;placement:</string>
          </property>
          <property name="buddy" >
           <cstring>m_value</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_value" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_options" >
     <property name="title" >
      <string>Replacement Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="AutoReplaceIncoming" >
        <property name="text" >
         <string>Auto replace on incoming messages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AutoReplaceOutgoing" >
        <property name="text" >
         <string>Auto replace on outgoing messages</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DotEndSentence</tabstop>
  <tabstop>CapitalizeBeginningSentence</tabstop>
  <tabstop>m_key</tabstop>
  <tabstop>m_value</tabstop>
  <tabstop>m_list</tabstop>
  <tabstop>m_add</tabstop>
  <tabstop>m_edit</tabstop>
  <tabstop>m_remove</tabstop>
  <tabstop>AutoReplaceIncoming</tabstop>
  <tabstop>AutoReplaceOutgoing</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
